src/app/App.js

```diff
@@ -36,6 +38,25 @@ class App extends Component {
     })
   }
 
+  checkForErrors = response => {
+    if (!response.ok) {
+      this.setState( {error: 'Oop, that/s no good 😅'} )
+    } else {
+      return response.json();
+    }
+  }
+
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(this.checkForErrors)
+      .then(data => {
+        this.setState( {
+          movies: data.movies
+        })
+      })
+      .catch(err => this.setState({ error: 'Oh wow. This is embarassing. Try again later? 😅' }))
+  }
+
```

Comment:
Nice work getting these network requests integrated so seamlessly. One thing to keep in mind: the React Architecture section of the rubric stipulates that API calls should be housed in their own separate file. Should be easy to migrate them over and import into the App component.

Reply:
Yeah - generally a good idea, and it shouldn't be a difficult refactor!